include/CoolPropLib.h

```diff
@@ -270,6 +270,17 @@
      * @return A handle to the state class generated
      */
     EXPORT_CODE long CONVENTION AbstractState_factory(const char* backend, const char* fluids, long *errcode, char *message_buffer, const long buffer_length);
+    /**
+     * @brief Get the fluid names for the AbstractState
+     * @param handle The integer handle for the state class stored in memory
+     * @param fluids The array of fluids
+     * @param N number of fluids
+     * @param errcode The errorcode that is returned (0 = no error, !0 = error)
+     * @param message_buffer A buffer for the error code
+     * @param buffer_length The length of the buffer for the error code
+     * @return 
+     */
+    EXPORT_CODE void CONVENTION AbstractState_fluid_names(const long handle, char **& fluids, long *N, long *errcode, char *message_buffer, const long buffer_length);
```

Comment:
Please return a delimited string in a pre-allocated buffer passed into the function.  See other examples that return strings in this file.